### PR TITLE
Fix for Inline block doesn't render in captions

### LINKED ISSUE #4744: Inline block doesn't render in captions
**Describe the bug**
The inline block doesn't render in captions. This is true of image captions and media block captions. Markup is present in Optimo and Ares

**To Reproduce**

1. Go to https://www.test.bbc.co.uk/news/articles/c2054qjjzjdo
2. Scroll to the first block of text. See that inline block renders correctly and is marked up as 
`<span lang="fa" class="InlineSpan-sc-1w2gf2n-0 ffibHJ">چرا می‌توانید به بی‌بی‌سی اطمینان کنید؟</span>`
 (this is the correct language markup for this inline block of Farsi text)
3. Scroll down to the image. See that the image caption does not render the inline block, and there is no span markup:
`<p>This image caption has an inline block: <!-- -->. See, what a lovely inline block.</p>`
4. Scroll down to the media block. See the same error again. (No inline block and no span markup).
5. Go to the Ares broker API data for this article (ask @lizcameron for a link to this) and see that image captions and media block captions are marked up as having inline block fragments in another language.

**Expected behavior**
Inline blocks in captions should be marked up as such and should be rendering on the page.

**Screenshots**

Markup present in Optimo:
image
Markup present in Ares:
image
Article page with no inline block markup for the image caption:
image

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome, Firefox

**Smartphone (please complete the following information):**

- Device: Pixel 2 
- OS: Android 10
- Browser: Chrome

Expected to be an issue on all browsers/devices but only tested on the above.

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: RayNjeri <rachael.njeri@andela.com>

Resolves  #4744

**Overall change:** _Adds inline link to Caption container to support inline links in captions._

Test asset --> `c2054qjjzjdo`

Screenshot 2019-11-28 at 11 00 44 AM
Screenshot 2019-11-28 at 11 00 32 AM
Screenshot 2019-11-28 at 11 00 25 AM

**Testing Notes:**
- Run the branch locally, Review this test asset `c2054qjjzjdo`, Noticed the inline line as shown in the screenshots also inspect element

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Caption/index.jsx [hunk 1]
```diff
@@ -7,8 +7,9 @@ import { ServiceContext } from '#contexts/ServiceContext';
 import Blocks from '../Blocks';
 import Fragment from '../Fragment';
 import InlineLink from '../InlineLink';
+import inline from '../InlineContainer';
```
Comment (EinsteinNjoroge) [inline src/app/containers/Caption/index.jsx]:
```suggestion
import Inline from '../InlineContainer';
```

File: src/app/containers/Caption/index.jsx [hunk 2]
```diff
@@ -7,8 +7,9 @@ import { ServiceContext } from '#contexts/ServiceContext';
...
 
-const componentsToRender = { fragment: Fragment, urlLink: InlineLink };
+const componentsToRender = { fragment: Fragment, urlLink: InlineLink, inline };
```
Comment (EinsteinNjoroge) [inline src/app/containers/Caption/index.jsx]:
```suggestion
const componentsToRender = { fragment: Fragment, urlLink: InlineLink, Inline };
```

**Review by EinsteinNjoroge — COMMENTED** [review]
Looks good, just a few changes 🙂.

- We need to add tests for this change.
- It would be useful to add a screenshot to this PR's description.

File: src/app/containers/Caption/index.jsx (hunk 1 above)
Comment (RayNjeri) [reply src/app/containers/Caption/index.jsx]: Has been implemented as `inline` throughout the code base maintained it as so for consistency https://github.com/bbc/simorgh/blob/9a5daab7ad8777aa2f0b9350440968f82abfce77/src/app/containers/Paragraph/index.jsx#L7

File: src/app/containers/Caption/index.jsx (hunk 2 above)
Comment (RayNjeri) [reply src/app/containers/Caption/index.jsx]: same as above https://github.com/bbc/simorgh/blob/9a5daab7ad8777aa2f0b9350440968f82abfce77/src/app/containers/Paragraph/index.jsx#L11

File: src/app/containers/Caption/index.jsx (hunk 1 above)
Comment (EinsteinNjoroge) [reply src/app/containers/Caption/index.jsx]: 🤔that must have been left in there by mistake, please update that too.

**Review by EinsteinNjoroge — APPROVED** [review]
Looks good :+1:

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Review by Bopchy — APPROVED** [review]

**Review by jamesdonoh — APPROVED** [review]
Looks good, thanks @RayNjeri. Please can you update the description? 'Overall change' is boilerplate and there is no list of code changes. Also please can you add some guidance for testing?

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**